### DISCUSSION TITLE
How can we collaborate on documentation?

### DISCUSSION THREAD

**Discussion by vascanera** [replit/kaboom#229] [General]
💥  Kaboom is insanely great! I / we love it! ❤️ 

But, as one can see clearly, the documentation is lacking a lot of (useful) details. 
How can we (github users) help?

For instance, one example comes to mind:  the documentation for time() is pretty vague.
```
time() => number
Get the total time since beginning.
```
Since the beginning of what?
What is the unit of time used? Is it seconds (as float)? Is it milliseconds?
This could be easily confusing unless specified in the documentation.

So, how can we help improve the docs? Writing docs is time consuming, we know that, and we can give a helping hand in return for all your great work.

Cheers!

**Comment by slmjkdbtl** [comment]
Thank you! I agree a lot of docs are under-quality now, and thanks for understanding! 

All the documentations are all in [`src/types.ts`](https://github.com/replit/kaboom/blob/master/src/types.ts) as jsdoc above each function. Feel free to add anything there! It'll be great help 🙏
